src/main/java/com/stage/users/serviceImpl/UserServiceImp.java

```diff
@@ -0,0 +1,58 @@
+package com.stage.users.ServiceImpl;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.stage.users.Iservice.UserIservice;
+import com.stage.users.Model.User;
+import com.stage.users.Repository.UserRepository;
+
+@Service
+public class UserServiceImp implements UserIservice{
```

Comment:
please call it UserServiceImpl within a folder of serviceImpl and this folder should exist within a service folder 
so service |- serviceImpl |-UserServiceImpl
UserServiceImpl should implement UserService